src/t_hash.c

```diff
@@ -2733,14 +2811,36 @@ int hfieldIsExpired(hfield field) {
 /*-----------------------------------------------------------------------------
  * Hash Field Expiration (HFE)
  *----------------------------------------------------------------------------*/
-/* Called during active expiration of hash-fields */
+static void propagateHashFieldDeletion(redisDb *db, sds key, char *field, size_t fieldLen) {
+    robj *argv[] = {
+            shared.hdel,
+            createStringObject((char*) key, sdslen(key)),
+            createStringObject(field, fieldLen)
+    };
+
+    enterExecutionUnit(1, 0);
```

Comment:
why do we put these into executeUnit?

Reply:
Bad copy paste of code (even though it makes no harm). Thanks!

Reply:
@sundb , I reviewed it again. I think it is harmless. Please review it.

Reply:
yes, it's harmless but it's also NOP, we only do the `alsoPropagate` here, and we don't notify any event to module or process any other commands.
please point out if i'm wrong.

Reply:
It can be invoked either by the activeExpireCycle() cron job or through an actual query from the client.